metadata.go

```diff
@@ -80,12 +82,36 @@ var debianLicenseText = map[string]string{
 `,
 }
 
+var githubRegexp = regexp.MustCompile(`github\.com/([\w._-]+/[\w._-]+)`)
```

Comment:
I don’t doubt that it works with the cases you’ve tested, but my suggestion is clearer, simpler and more reliable: after all, slashes are the path separators in URLs, so that’s what we should work with.